test_robotiq.ops

```diff
@@ -1,9 +1,11 @@
 import("soem_beckhoff_drivers")
 import("soem_robotiq_drivers")
-import("rtt_rosnode") // makes this process a ROS node
+import("soem_master")
+import("rtt_rosnode")
+import("rtt_roscomm")
 
 loadComponent("Master","soem_master::SoemMasterComponent")
-Master.ifname = "eth1" 
+Master.ifname = "enp9s0"
```

Comment:
Just as a side note, we may want to keep the ifname to something standard as eth0 or eth1 for the repository?